src/js/components/DetailViewSectionHeading.js

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import React from 'react';
+
+const DetailViewSectionHeading = (props) => {
+  const {children, className, level} = props;
+
+  return React.createElement(
+    `h${level}`,
+    {
```

Comment:
Let's extract a  variable like `headingProps` and then it should fit 80 chars line, so it'd me more readable